.github/actions/repo-gardening/src/get-affected-changelogger-projects.js

```diff
@@ -0,0 +1,60 @@
+const glob = require( 'glob' );
+const fs = require( 'fs' );
+
+/**
+ * Returns a list of Projects that use changelogger package
+ *
+ * @returns {Array} list of changelogger packages
+ */
+function getChangeloggerProjects() {
+	const projects = [];
+	const composerFiles = glob.sync( process.env.GITHUB_WORKSPACE + '/projects/*/*/composer.json' );
+	composerFiles.forEach( file => {
+		const json = JSON.parse( fs.readFileSync( file ) );
+		if (
+			! file.includes( 'changelogger' ) &&
```

Comment:
I'm not sure I understand this. Do we need to check for this if we look for it in the require below?

Reply:
Ah, ok, I see what you were trying to do there. The `'projects/packages/changelogger/composer.json' !== $file` bit in there is because projects/packages/changelogger/composer.json doesn't `require` or `require-dev` automattic/jetpack-changelogger because it *is* automattic/jetpack-changelogger.

Reply:
Hm, I get confused at first 😬 

Here I looping through all the `composer.json` files in the monorepo to find these projects which use `changelogger` package. Also, we should include `changelogger` project itself.